near-sdk/tests/code_size.rs

```diff
@@ -0,0 +1,37 @@
+/// Compiles contract to wasm with release configuration and returns the code size.
+fn check_example_size(example: &str) -> usize {
+    let status = std::process::Command::new("cargo")
```

Comment:
I'd add `Command::new("rustup").args(&["target", "add", "wasm32-unknown-unknown"]).status().unwrap()` here, so that the test *just works* in more cases.

Reply:
hmm, that seems like a bad implication to update a user's toolchain unknowingly through running a test, maybe if this test was ignored by default?

Reply:
yeah, that's a fair point. I'll make this change